rskj-core/src/test/java/co/rsk/peg/bitcoin/UtxoUtilsTest.java

```diff
@@ -168,4 +182,75 @@ private static Stream<Arguments> invalidEncodedOutpointValues() {
 
         return arguments.stream();
     }
+
+    @Test
+    void extractOutpointValues_validBtcTransaction_shouldReturnOutpointValues() {
+        Coin amountToSend = Coin.COIN;
+
+        BtcTransaction fundingTransaction = new BtcTransaction(btcMainnetParams);
+        fundingTransaction.addInput(
+            BitcoinTestUtils.createHash(1),
+            FIRST_OUTPUT_INDEX,
+            new Script(new byte[]{})
+        );
+        fundingTransaction.addOutput(amountToSend, TEST_ERP_FEDERATION.getAddress());
+
+        BtcTransaction pegout = new BtcTransaction(btcMainnetParams);
+        pegout.addInput(fundingTransaction.getOutput(FIRST_OUTPUT_INDEX));
+        pegout.addOutput(amountToSend, testAddress);
```

Comment:
Do you really need all this setup? Looks like all you really need is a Bitcoin transaction with some inputs

Reply:
The inputs must have the outpoints because the outpoint value is obtained from them. So, at least the funding transactions are needed. I can remove the outputs but we will be testing the method against an unrealistic transaction. I prefer it as it is now.